openapi/schemas/processes-core/statusInfo.yaml

```diff
@@ -1,42 +1,48 @@
-type: object
-required:
-   - id
-   - status
-   - type
-properties:
-   id:
-      type: string
-   processID:
-      type: string
-   type:
-      type: string
-      example:
-        - process
-        - wps
-        - openeo
-   status:
-      $ref: "statusCode.yaml"
-   message:
-      type: string
-   exception:
-      $ref: "../common-core/exception.yaml"
-   created:
-      type: string
-      format: date-time
-   started:
-      type: string
-      format: date-time
-   finished:
-      type: string
-      format: date-time
-   updated:
-      type: string
-      format: date-time
-   progress:
-      type: integer
-      minimum: 0
-      maximum: 100
-   links:
-      type: array
-      items:
-         $ref: "../common-core/link.yaml"
+allOf:
+  - $ref: "./descriptionType.yaml"
+  - type: object
+    required:
+      - id
+      - status
+      - type
+    properties:
+      id:
+        type: string
+      processID:
+        type: string
```

Comment:
Similarly to previous comment. The job's process might not reside on this server. Therefore, `process: <uri>` is preferable. Consider also that `process` might be an object at some point (openEO alignment), similar to the `request` property below.

Reply:
Done.